verify.js

```diff
@@ -57,15 +64,15 @@ class Verify {
         .then(res => {
           if (res.status !== 200) {
             debug(`${buildId} | Fail ${uri}: ${res.status}`);
-            return next(null, uri);
+            return next(null, { buildId, uri, reason: `status ${res.status}` });
           }
 
           debug(`${buildId} | Fetch ok ${uri}`);
           next();
         })
         .catch(err => {
           debug(`${buildId} | Fail ${uri}: ${err}`);
-          return next(null, uri);
+          return next(null, { buildId, uri, reason: err });
```

Comment:
`err` here would be an Error object right? Should you stringify it in some way? `err && err.toString() || 'unknown'` or just `''+err`?